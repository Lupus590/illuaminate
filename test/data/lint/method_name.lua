-- config: (lint (only (syntax:method-name)))

function x:y:z() end
function x:y.z() end
function x.y:z() end --ok
