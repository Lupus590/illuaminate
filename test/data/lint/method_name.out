method_name.lua:[3:11-3:12]: Method names cannot appear at this position. [syntax:method-name]
   │
3  │ function x:y:z() end
   │            ^
method_name.lua:[4:11-4:12]: Method names cannot appear at this position. [syntax:method-name]
   │
4  │ function x:y.z() end
   │            ^
Compilation failed with 2 errors and 0 warnings
